### PR TITLE
Added parameter SkipPackageValidation to ServiceFabricDeploy

### PR DESCRIPTION
**Pull request by mikanyg**
- Exposes the script parameter to skip package validation as requested in #3742

### CODE REVIEW

**Comment by mikanyg** [comment]
@MikkelHegn @NCarlsonMSFT Had 30 minutes I could spare ;-)

**Comment by NCarlsonMSFT** [comment]
Have you deployed and validated the change?

**Comment by mikanyg** [comment]
Yes I have

**Comment by NCarlsonMSFT** [comment]
@kmkumaran, @sachinma, @bishalpd any objections to merging this?

File: Tasks/ServiceFabricDeploy/task.json [hunk 1]
```diff
@@ -119,6 +119,15 @@
             "helpMarkDown": "Indicates whether an upgrade will be skipped if the same application type and version already exists in the cluster, otherwise the upgrade fails during validation. If enabled, re-deployments are idempotent."
         },
         {
+            "name": "skipPackageValidation",
+            "type": "boolean",
+            "label": "Skip package validation",
```
Comment (bishal-pdMSFT) [inline Tasks/ServiceFabricDeploy/task.json]: Should we be more specific and say "Skip application package validation".

**Comment by bishal-pdMSFT** [comment]
Looks good to me.

File: Tasks/ServiceFabricDeploy/task.json (hunk 1 above)
Comment (mikanyg) [inline Tasks/ServiceFabricDeploy/task.json]: Long labels messes with the UI and its quite long as it is. The details should rather be in the helpMarkDown, which sadly is not being used enough for other "older" parameters of this task.

**Pull request merged**